MAINTAINERS.md

```diff
@@ -4,12 +4,12 @@ Fluent Bit is developed and supported by many individuals and companies.  The fo
 
 | Maintainer Name                                       | Components               | Company                                           |
 | :---------------------------------------------------- | ------------------------ | ------------------------------------------------- |
-| [Eduardo Silva](https://github.com/edsiper)           | All                      | [Calyptia](https://calyptia.com)                  |
-| [Leonardo Alminana](https://github.com/leonardo-albertovich) | All               | [Calyptia](https://calyptia.com)                  |
+| [Eduardo Silva](https://github.com/edsiper)           | All                      | [Chronosphere](https://chronosphere.io)                  |
+| [Leonardo Alminana](https://github.com/leonardo-albertovich) | All               | [Chronosphere](https://chronosphere.io)                  |
 | [Masoud Koleini](https://github.com/koleini)          | Stream Processor         | [Arm](https://www.arm.com/)                       |
-| [Hiroshi Hatake](https://github.com/cosmo0920)        | All                      | [Calyptia](https://calyptia.com)                  |
+| [Hiroshi Hatake](https://github.com/cosmo0920)        | All                      | [Chronosphere](https://chronosphere.io)                  |
 | [Fujimoto Seiji](https://github.com/fujimotos)        | Windows Platform         | Individual                                        |
 | [Wesley Pettit](https://github.com/PettitWesley)      | Amazon Plugins (AWS)     | [Amazon Web Services](https://aws.amazon.com/)    |
 | [Cedric Lamoriniere](https://github.com/clamoriniere) | Datadog Output Plugin    | [Datadog](https://www.datadoghq.com/)             |
 | [Jonathan Gonzalez V.](https://github.com/sxd)        | PostgreSQL Output Plugin | [2ndQuadrant](https://www.2ndquadrant.com/en/)    |
-| [Jorge Niedbalski](https://github.com/niedbalski)     | CI && Containers         | [Calyptia](https://calyptia.com/)                 |
+| [Jorge Niedbalski](https://github.com/niedbalski)     | CI && Containers         | [Chronosphere](https://chronosphere.io/)                 |
```

Comment:
https://niedbalski.dev

Reply:
These seem to be company names for all the others and it was Calyptia before.

Reply:
I think is up to each contributor.